base/gfspconfig/config.go

```diff
@@ -222,6 +222,8 @@ type ParallelConfig struct {
 
 	LoadReplicateTimeout int64 `comment:"optional"`
 	LoadSealTimeout      int64 `comment:"optional"`
+
+	SyncAvailableVGFInterval int `comment:"optional"`
```

Comment:
You can directly hardcode it in the code using macros.
Exporting as a config will increase the cognitive and maintenance costs of the SP.

Reply:
fixed with default value 5 mins